pkgs/tools/misc/wlc/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, argcomplete
+, python-dateutil
+, requests
+, pyxdg
+, pre-commit
+, pytest
+, pytest-cov
+, responses
+, twine
+}:
+
+buildPythonPackage rec {
+  pname = "wlc";
+  version = "1.11";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256:0ysx250v2qycy1m3jj0wxmyf2f5n8fxf6br69vcbyq2cnqw609nx";
+  };
+
+  propagatedBuildInputs = [ argcomplete python-dateutil requests pyxdg pre-commit pytest pytest-cov responses twine ];
```

really long, testing inputs should go in checkInputs
```suggestion
  propagatedBuildInputs = [
    argcomplete
    python-dateutil
    requests
    pyxdg
    pre-commit
    responses
    twine
  ];
    
    checkInputs = [
      pytestCheckHook
      pytest-cov
   ]
```

---

also, preference for `pytestCheckHook` usage. https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pytestcheckhook